src/components/donation-flow/steps/payment-method/PaymentMethod.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext } from 'react'
```

Comment:
PaymentMethod from the root of `steps` was moved to here, but git recognizes is a new file